frontend/.env

```diff
@@ -11,6 +11,7 @@ VITE_ENDPOINT_SUPPLIER=stockView/supplier?ownMaterialNumber=
 VITE_ENDPOINT_REPORTED_MATERIAL_STOCKS=stockView/reported-material-stocks?ownMaterialNumber=
 VITE_ENDPOINT_REPORTED_PRODUCT_STOCKS=stockView/reported-product-stocks?ownMaterialNumber=
 VITE_ENDPOINT_UPDATE_REPORTED_MATERIAL_STOCKS=stockView/update-reported-material-stocks?ownMaterialNumber=
+VITE_ENDPOINT_UPDATE_REPORTED_PRODUCT_STOCKS=stockView/update-reported-product-stocks?ownMaterialNumber=
```

Comment:
We also need to update to further files, to add the endpoint to the environment variables:
- frontend/src/config.json 
- frontend/.env.dockerbuild
There is an explanation in file docs/DEVELOPMENT.md

Reason is that vite always hard codes environment variables when building (which is done in docker). We use a workaround to write variables into the built code and then substring these with environment variables via a script. In case of any questions, feel free to ask :)

Currently it works because this .env is the default value (not overwritten by .env.dockerbuild), but we don't have the opportunity to set it via environment.

Reply:
Done